client/src/NetworkClient.ts

```diff
@@ -1,39 +1,52 @@
 import { EventClient, EventCallback } from '@nimiq/rpc-events';
 
-export type PlainTransaction = {
-    sender: string,
-    senderPubKey: Uint8Array,
-    recipient: string,
-    value: number,
-    fee: number,
-    validityStartHeight: number,
-    signature: Uint8Array,
-    extraData?: string | Uint8Array,
+export interface PlainTransaction {
```

Comment:
Note: although tslint complains because of rules, I see no reason why we have to use interface here. The only advantage of interface is that it can be merged with existing definitions, which is useful for extending third-party libraries or built-in types (like HtmlElement or window).

Reply:
(So it could be relevant for third-party devs in theory. I don't think it makes sense here.)

Reply:
+1 for the comments explaining the values, but I don't know if changing from type to interface is necessary.